cmd/xl-v1-utils.go

```diff
@@ -162,6 +162,11 @@ func parseXLErasureInfo(ctx context.Context, xlMetaBuf []byte) (ErasureInfo, err
 	erasure.Index = int(erasureResult.Get("index").Int())
 
 	checkSumsResult := erasureResult.Get("checksum").Array()
+
+	if len(checkSumsResult) == 0 {
```

Comment:
logger.LogIf here

Reply:
We will not have enough information here, that is why I am logging at a higher level.